docs/openapi/pipeline.yaml

```diff
@@ -64,6 +64,24 @@ paths:
               schema:
                 $ref: '#/components/schemas/ListCatalogResponse'
 
+  '/api/v1/orgs/{orgId}/catalogs/update':
```

Comment:
Please run `make generate-client` to add this to the Go client (also add a `operationId` field)